### ISSUE TITLE
View Proposal For Channel Payouts

### ISSUE THREAD

**Issue by dmtrjsg** [Joystream/pioneer#3074]
### Context

Same as in 
-  #3073 

### Scope

As a Council Member I want to view Channel Payout Proposal

Acceptance criteria:

Specific parameters shown as widgets showing key info for decision makers: 

in scope:
- [ ] Show Max and Min Reward amount/  Max and Min Cashout
- [ ] Block/ Unblock all cashout attempts (e.g. cashouts - allowed)

out of scope:

- [ ] Hash of payload
- [ ] "Download the file" from widget
- [ ] Credit WG budget

**Comment by MattAugustynowicz** [comment]
[👉 Figma link](https://www.figma.com/file/GlgN8uBRtvtMJtiOsdtDF7/Pioneer-Design?node-id=9707%3A376124)

![image.png](https://images.zenhubusercontent.com/61f04aadb8abb13c0fc2e178/6d342c57-83bf-4782-b517-0f7d1aecfa98)

[👉 Blocked Payout Possibility here](https://www.figma.com/file/GlgN8uBRtvtMJtiOsdtDF7/Pioneer-Design?node-id=9707%3A376321)

![image.png](https://images.zenhubusercontent.com/61f04aadb8abb13c0fc2e178/d1211648-e5f5-41da-8805-03aa9913e153)

**Comment by dmtrjsg** [comment]
Additions:

Add new widgets for Proposal Payout Preview:
- [ ] Data Object ID  
+ Tootlip:
This is the ID submitted to Chain for the Data Object (payout payload) to be further uploaded to the Storage. 

- [ ] Payload Hash 
+ tooltip
"Payload file hash generated at the point of converting the Vector File containing human readable payout information to the Executable payload file."

- [ ] Payload Size
"Size of payload that is submitted to the chain in the proposal extrensic."

@thesan pls review the tooltips content before implementation 😇

**Comment by dmtrjsg** [comment]
@bedeho size is not added to QN now, do we need to add it, or hash is enough?

**Comment by bedeho** [comment]
Not needed or launch, but add it as a future enhancement.
